### DISCUSSION TITLE
Most links in Pluto TV not working in Kodi

### DISCUSSION THREAD

**Discussion by jkemp814** [iptv-org/community#202] [Show and tell]
Can someone check if Pluto TV links are working on their end. While browsing down the list, most are not working in Kodi. 

Tried in Firefox, and at first they weren't working there either. Tried more down the list, and they suddenly started loading. I see a lot of the channels must have passed the resolution scanner.

**Comment by Dum4G** [comment]
I've noticed that some are infinitely loading in VLC quite recently. Probably has something to do with that bunch of parameters that are used after the stream link. Will require an investigation and batch edition

**Comment by Sphinxroot** [comment]
everything is fine in kodi on my side (m3u)  , but  you might wanna try  Matt Huismans (SlyGuy) addons EPG is included

**Reply by jkemp814** [reply]
Okay, thanks for the lead.
